homeassistant/components/renson/services.yaml

```diff
@@ -0,0 +1,154 @@
+set_manual_level:
```

Comment:
Can it be a `Select` entity?

Reply:
Removed this because it is already been replaced by fan entity